rustio/src/client.rs

```diff
@@ -82,15 +82,11 @@ impl Client {
         Self::send_get_request(url).unwrap().json().unwrap()
     }
 
-    pub fn get_station_by_id(&self, id: i32) -> Result<Station,&str> {
+    pub fn get_station_by_id(&self, id: i32) -> Option<Station> {
         let url = format!("{}{}{}", BASE_URL, STATION_BY_ID, id);
         let mut result : Vec<Station> = Self::send_get_request(url).unwrap().json().unwrap();
 
-        if result.len() > 0 {
-            Ok(result.remove(0))
-        }else {
-            Err("ID points to an empty station")
-        }
+        result.pop()
```

Comment:
*Normally* there should be always a station available to return. And if not, it should be definitely correctly handled by the caller, because this behavior is not indented/wanted.